parm/config/gfs/config.wave

```diff
@@ -25,26 +25,60 @@ export CDUMPwave="${RUN}wave"
 export CDUMPRSTwave="gdas"
 
 # Grids for wave model
-# GFSv16
-export waveGRD=${waveGRD:-'gnh_10m aoc_9km gsh_15m'}
-export waveGRDN=${waveGRDN:-'1 2 3'} # gridnumber for ww3_multi
-export waveGRDG=${waveGRDG:-'10 20 30'} # gridgroup for ww3_multi
-export USE_WAV_RMP=${USE_WAV_RMP:-'YES'} #yes/no rmp grid remapping pre-processed coefficients
-export waveMULTIGRID=${waveMULTIGRID:-'.true.'}
-export MESH_WAV=${MESH_WAV:-'mesh.gwes_30m.nc'}
-
-# ESMF input grid
-export waveesmfGRD=${waveesmfGRD:-'glox_10m'} # input grid
+export waveGRD=${waveGRD:-'mx025'}
+
+#Grid dependent variables for various grids
+case "${waveGRD}" in
+  "gnh_10m;aoc_9km;gsh_15m")
+    #GFSv16 settings: 
+    waveGRDN='1 2 3'
+    waveGRDG='10 20 30'
+    USE_WAV_RMP='YES'
+    waveMULTIGRID='.true.'
+    IOSRV='3' 
+    MESH_WAV=' '
+    waveesmfGRD='glox_10m'
+    waveuoutpGRD='points'
+    waveinterpGRD='glo_15mxt at_10m ep_10m wc_10m glo_30m'
+    wavepostGRD='gnh_10m aoc_9km gsh_15m'
+    ;;
+  "gwes_30m")
+    #Grid used for P8
+    waveinterpGRD=' '
+    wavepostGRD='gwes_30m'
+    ;;
+  "mx025")
+    #Grid used for HR1 (tripolar 1/4 deg)
+    waveinterpGRD='reg025'
+    wavepostGRD=' '
+    ;;
+  "glo_025") 
+    #GEFSv13 regular lat/lon 1/4 deg grid 
+    waveGRD='glo_025'
+    waveinterpGRD=' '
+    wavepostGRD='glo_025'
+    ;;
+esac
+
+
+export waveGRD=${waveGRD:-'mx025'}
+export waveGRDN=${waveGRDN:-'1'} # gridnumber for ww3_multi
+export waveGRDG=${waveGRDG:-'10'} # gridgroup for ww3_multi
```

Comment:
Lines 34 and 35 set these for `gnh_10m;aoc_9km;gsh_15m`.

Probably better to move these and the above line before the `case` block.

Reply:
@aerorahul   So my thought process for this is that the values will be 1, 10 for anything except gnh_10m;aoc_9km;gsh_15m so I put the exception in the case block and then everything else is this.  Do you want this defined for every single grid?  Also I didn't do the "export" in any of the case blocks, so I guess that would also need to be added if this is moved?

Reply:
@WalterKolczynski-NOAA Can you point me to a piece of code where I could see this approach?

Reply:
@JessicaMeixner-NOAA in config.base, where we set the settings based on `$APP`. We set all the `DO_<Component>` to "NO", then change appropriate ones to "YES" based on the APP.